Adds Showers to Engie

LastCrusader105 opened the pull request:
deletes and adds new showers away from blast doors, moves fire closets to make room for showers

And adds showers to Pubby Engie and one to Atmos

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
stops players getting crushed by blast doors, and restores my honour

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: Moved the engineering shower heads outside of the blast doors
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Commit:
Adds Showers to box, moves fire closets

deletes and adds new showers away from blast doors, moves fire closets to make room for showers

LastCrusader105 commented:
@qwertyquerty cheers for the help, it is now done

LastCrusader105 commented:
@Crossedfall it’s ready to merge

Commit:
adds showers to Pubby Engie And atmos

as above

Commit:
removes the random shower i added in space

PowerfulBacon commented:
Those lockers are in a pretty bad position now

LastCrusader105 commented:
> Those lockers are in a pretty bad position now

Thanks for feedback
Here’s the thing @PowerfulBacon, one fire suits are pretty much pointless with fastmos, Sm and surrounding area gets too hot too quick
And two, where else can they go? They go inside Sm area they are useless, outside they have some uses.
Commenting “bad place” doesn’t help
Where do you think they should go?

As for pubby it’s just 3 showers nothing moved etc

LastCrusader105 commented:
![lockers](https://user-images.githubusercontent.com/56616002/97093923-d495ac80-1647-11eb-89d2-5ad1245d7c3f.png)

heres where lockers currently are

only place i could see working would be here but it would leave you with a 1 tile "walk way"

![lockers](https://user-images.githubusercontent.com/56616002/97093963-276f6400-1648-11eb-898b-71c373936f24.png)

@PowerfulBacon i think i got it

![lockers](https://user-images.githubusercontent.com/56616002/97094006-ac5a7d80-1648-11eb-9b4b-7bef7b8de473.png)
replace the engie lockers with fireclosets and move engie ones to where ive indicated

LastCrusader105 commented:
@BeloneX you May give your thoughts
Also please teach how to ask for reviewer cheers

LastCrusader105 commented:
sadly I dunno how to request a review so @PowerfulBacon @Crossedfall @ike709 @qwertyquerty 

its good to go unless you got more feedback than bacons "no good"
cheers

Crossedfall commented:
I think your second screenshot would be better for locker placement. You could also replace the plant in the lower right with an engineering locker, instead of having the locker in the actual walkway.

LastCrusader105 commented:
![firecloset](https://user-images.githubusercontent.com/56616002/97350518-891e1100-1888-11eb-8e1b-d91ddb90983a.png)


moved the engie lockers cause they arnt important and replaced with fire closets

2 mins while i push it

Commit:
moved lockers about

LastCrusader105 commented:
@Crossedfall we are good to go, also please explain how you guys run this, do you check it daily or do i have to @ you guys when its good to go etc?

also if it could be pushed though quick id like to as bee babies have also added it and its incomplete

LastCrusader105 commented:
@PowerfulBacon your head dev now?
shouldn’t be a 12 day review
Cheers 😘

PowerfulBacon reviewed (approved):
Yea seems hood, sorry about the delay
